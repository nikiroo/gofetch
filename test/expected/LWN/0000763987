                      KERNEL PREPATCH 4.19-RC2                     

  [Kernel] Sep 2, 2018 22:29 UTC (Sun) (corbet)

  o Reference: 0000763987
  o News link: https://lwn.net/Articles/763987/
  o Source link: 


    The  [1]4.19-rc2  kernel  prepatch  is  out  for  testing. " As
    usual,  the  rc2  release  is pretty small. People are taking a
    breather  after  the  merge  window, and it takes a bit of time
    for bug reports to start coming in and get identified. "
    
    
    
    [1] https://lwn.net/Articles/763988/



