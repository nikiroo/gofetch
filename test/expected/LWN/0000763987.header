0Kernel prepatch 4.19-rc2	null/LWN/0000763987		70
i  [Kernel] Sep 2, 2018 22:29 UTC (Sun) (corbet)
i
i    The  [1]4.19-rc2  kernel  prepatch  is  out  for  testing. " As
i    usual,  the  rc2  release  is pretty small. People are taking a
i    breather  after  the  merge  window, and it takes a bit of time
i    for bug reports to start coming in and get identified. "
i    
i    
i    
i    [1] https://lwn.net/Articles/763988/
i
