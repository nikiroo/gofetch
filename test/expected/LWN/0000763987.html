<!DOCTYPE html>
<html>
<head>
  <meta http-equiv='content-type' content='text/html; charset=utf-8'>
  <meta name='viewport' content='width=device-width, initial-scale=1.0'>
  <style type='text/css'>
    body { margin: 1em 15%; }
  </style>
</head>
<body>
<div class='story'>
	<h2>Kernel prepatch 4.19-rc2</h2>
	<div class='details'>([Kernel] Sep 2, 2018 22:29 UTC (Sun) (corbet))</div>
	<br/>
    <ul>
    <li>Reference: <a href=''>0000763987</a></li>
        <li>News link: <a href='https://lwn.net/Articles/763987/'>https://lwn.net/Articles/763987/</a></li>
        <li>Source link: <a href=''></a></li>
    </ul>
	<br/>
	<div class='content' style='text-align: justify'>
		The [1]4.19-rc2 kernel prepatch is out for testing. &quot; As usual, the rc2 release is pretty small. People are taking a breather after the merge window, and it takes a bit of time for bug reports to start coming in and get identified. &quot;<br/><br/><br/><br/>[1] https://lwn.net/Articles/763988/
	</div>
<hr/>
</div>
</body>
